Freezing layer 'model.23.dfl.conv.weight'
[34m[1mAMP: [39m[22mrunning Automatic Mixed Precision (AMP) checks with YOLOv8n...
[34m[1mAMP: [39m[22mchecks passed âœ…
[34m[1mtrain: [39m[22mScanning /home/animated/Projects/Sailbot/src/object_detection/object_detection/datasets/SailbotVT-4/train/labels.cache... 12816 images, 1474 backgrounds, 0 corrupt
[34m[1mval: [39m[22mScanning /home/animated/Projects/Sailbot/src/object_detection/object_detection/datasets/SailbotVT-4/valid/labels.cache... 1845 images, 211 backgrounds, 0 corrupt: 10
[34m[1malbumentations: [39m[22mBlur(p=0.01, blur_limit=(3, 7)), MedianBlur(p=0.01, blur_limit=(3, 7)), ToGray(p=0.01), CLAHE(p=0.01, clip_limit=(1, 4.0), tile_grid_size=(8, 8))
Plotting labels to runs/detect/train11/labels.jpg...
[34m[1moptimizer:[39m[22m 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically...
[34m[1moptimizer:[39m[22m AdamW(lr=0.002, momentum=0.9) with parameter groups 185 weight(decay=0.0), 198 weight(decay=0.00046875), 197 bias(decay=0.0)
[34m[1mTensorBoard: [39m[22mmodel graph visualization added âœ…
Image sizes 640 train, 640 val
Using 8 dataloader workers
Logging results to [1mruns/detect/train11
Starting training for 4 epochs...
      Epoch    GPU_mem     box_om     cls_om     dfl_om     box_oo     cls_oo     dfl_oo  Instances       Size

        1/4        13G      1.319      3.524      1.399      1.399      5.854      1.213         49        640:   0%|          | 1/1282 [00:22<8:04:01, 22.67s/it]Exception ignored in: <generator object tqdm.__iter__ at 0x7f196dd1fed0>
Traceback (most recent call last):
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/std.py", line 1196, in __iter__
    self.close()
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/std.py", line 1302, in close
    self.display(pos=0)
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/std.py", line 1495, in display
    self.sp(self.__str__() if msg is None else msg)
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/std.py", line 459, in print_status
    fp_write('\r' + s + (' ' * max(last_len[0] - len_s, 0)))
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/std.py", line 452, in fp_write
    fp.write(str(s))
  File "/home/animated/.local/lib/python3.10/site-packages/tqdm/utils.py", line 196, in inner
    return func(*args, **kwargs)
KeyboardInterrupt:
Error in sys.excepthook:
Traceback (most recent call last):
  File "/home/animated/miniforge3/lib/python3.10/linecache.py", line 46, in getlines
    return updatecache(filename, module_globals)
  File "/home/animated/miniforge3/lib/python3.10/linecache.py", line 137, in updatecache
    lines = fp.readlines()
  File "/home/animated/miniforge3/lib/python3.10/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
KeyboardInterrupt
Original exception was:
Traceback (most recent call last):
  File "/home/animated/Projects/Sailbot/src/object_detection/object_detection/train_buoy_classifier.py", line 4, in <module>
    model.train(
  File "/home/animated/miniforge3/lib/python3.10/site-packages/ultralytics/engine/model.py", line 657, in train
    self.trainer.train()
  File "/home/animated/miniforge3/lib/python3.10/site-packages/ultralytics/engine/trainer.py", line 213, in train
    self._do_train(world_size)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/ultralytics/engine/trainer.py", line 389, in _do_train
    self.scaler.scale(self.loss).backward()
  File "/home/animated/miniforge3/lib/python3.10/site-packages/torch/_tensor.py", line 487, in backward
    torch.autograd.backward(
  File "/home/animated/miniforge3/lib/python3.10/site-packages/torch/autograd/__init__.py", line 200, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
KeyboardInterrupt